common/src/worldgen.rs

```diff
@@ -695,6 +695,7 @@ mod test {
                     state
                 },
                 chunks: Chunks::default(),
+                entities: Vec::new(),
```

Comment:
Is this redundant with the `graph_entities.rs` module? If not, we should document the purpose of both versions of the node -> entity mapping.

Reply:
Forgot that existed. Swapped in.